### PR TITLE
Massive Bookshelf Sync

### PR DESCRIPTION
**Pull request by Darkhax**

### CODE REVIEW

File: src/main/java/com/mcmoddev/lib/client/gui/GuiButtonGraphical.java [hunk 1]
```diff
@@ -14,7 +12,6 @@
  *
  * @author Tyler Hancock (Darkhax)
  */
-@SideOnly(Side.CLIENT)
```
Comment (adrwa) [inline src/main/java/com/mcmoddev/lib/client/gui/GuiButtonGraphical.java:17]: This should be side only

File: src/main/java/com/mcmoddev/lib/client/gui/GuiButtonGraphical.java [hunk 2]
```diff
@@ -35,44 +32,37 @@
...
-     *         The Y position for the button.
-     * @param texture
-     *         The image to be drawn on the button. This should be 20x20.
+     * @param id The id to assign the button. This id is specific to the GUI instance.
+     * @param x The X position for the button.
+     * @param y The Y position for the button.
+     * @param texture The image to be drawn on the button. This should be 20x20.
      */
     public GuiButtonGraphical(int id, int x, int y, ResourceLocation texture) {
+
```
Comment (adrwa) [inline src/main/java/com/mcmoddev/lib/client/gui/GuiButtonGraphical.java:44]: Can we do something about the formatting

**Review by adrwa — CHANGES_REQUESTED** [review]

File: src/main/java/com/mcmoddev/lib/client/gui/GuiButtonGraphical.java (hunk 1 above)
Comment (Darkhax) [reply src/main/java/com/mcmoddev/lib/client/gui/GuiButtonGraphical.java:17]: Button is now client side again.

File: src/main/java/com/mcmoddev/lib/client/gui/GuiButtonGraphical.java (hunk 2 above)
Comment (Darkhax) [reply src/main/java/com/mcmoddev/lib/client/gui/GuiButtonGraphical.java:44]: Will do in seperate PR to avoid merge conflicts.

**Review by adrwa — APPROVED** [review]

**Pull request merged**